app/backend/controllers/profile.js

```diff
@@ -7,12 +7,14 @@ const {findUserFollows, checkPassword, checkIBAN, checkTCKN} = require('../utils
   Get user id from parameter and responses accordingly.
 */
 
-async function profileResponse(user, me, followStatus, TradingEqFollow, Article) {
+async function profileResponse(user, me, followStatus, TradingEqFollow, Article, Portfolio) {
   let followings = await findUserFollows({ FollowingId: user._id, status: true })
   let followers = await findUserFollows({ FollowedId: user._id, status: true })
   let followRequests = await findUserFollows({ FollowedId: user._id, status: false})
   let followingTradings = await TradingEqFollow.find({ UserId : user._id })
   let articles = await Article.find().where('userId').equals(user._id)
+  let portfolios = await Portfolio.find().where('userId').equals(user._id)
```

Comment:
we can also check public control here, but if I am looking my own portfolios I have to see all portfolios no matter it is public or not.